wordpress-mobile/gutenberg-mobile
# Button block: Add width selector

Add a width selector to the button block. 

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/28543

To test:
Load this PR.
Add a button block.
Change the button width in settings.
Does it work as expected?

PR submission checklist:
